### PR TITLE
Qwk support

### PR DESCRIPTION
**Pull request by NuSkooler**
Initial QWK support exposed only with `oputil.js` until some additional testing occurs.

### CODE REVIEW

File: core/qwk_mail_packet.js [hunk 1]
```diff
@@ -279,13 +313,54 @@ class QWKPacketReader extends EventEmitter {
...
+
+            const FTNKludgeMapping = {
+                'X-FTN-PATH'    : 'PATH',
+                'X-FTN-MSGID'   : 'MSGID',
+                'X-FTN-REPLY'   : 'REPLY',
+                'X-FTN-PID'     : 'PID',
+                'X-FTN-FLAGS'   : 'FLAGS',
+                'X-FTN-TID'     : 'TID',
+                'X-FTN-CHRS'    : 'CHRS',
+                //  :TODO: X-FTN-KLUDGE - not sure what this is?
```
Comment (rswindell) [inline core/qwk_mail_packet.js:519]: X-FTN-KLUDGE is for otherwise-unrecognized/unsupported FTN control lines.

Comment (NuSkooler) [reply core/qwk_mail_packet.js:519]: @rswindell Thanks! What is the format?

Comment (rswindell) [reply core/qwk_mail_packet.js:519]: The contents (value) of the header field is everything found between (and not including) the Ctrl-A character and the carriage-return. e.g. X-FTN-KLUDGE: SOMEKLUDGE
Often the kludge will have a colon-terminated field name, not always.

Comment (NuSkooler) [reply core/qwk_mail_packet.js:519]: @rswindell sorry for all the questions - I guess multiple `X-FTN-KLUDGE` entries are allowed?

Comment (rswindell) [reply core/qwk_mail_packet.js:519]: Yes, multiple would be allowed. It's my way of retaining all the metadata of the original message and being able to reproduce it, even if I don't recognize it.

**Pull request merged**